src/components/AccountLogoBackground/AccountLogoBackground.tsx

```diff
@@ -0,0 +1,16 @@
+// AccountLogoBackground.tsx
+
+import { BiFile } from 'react-icons/bi'
+
+const AccountLogoBackground = (): JSX.Element => (
```

Comment:
```suggestion
const AccountLogoBackground:FC = () => (
```
这个naming 需要改一下，组件功能很模糊但看name